src/fields/Search.tsx

```diff
@@ -161,31 +179,81 @@ export function AutoComplete({
             onChange={handleChange}
             onSelect={handleSelect}
             open={isOpen}
+            renderMenuItem={(itemLabel, item: ItemDataType<T>) =>
+              MenuItem ? <MenuItem item={item.value} /> : itemLabel
+            }
+            value={inputValue}
             {...originalProps}
           />
         )}
+        {inputValue && (
+          <>
+            <Close
```

Comment:
For `Close` and `SearchIcon`, there is an `IconButton` component and I would `styled()` them rather than inline-styling them here.

Reply:
- `IconButton` is hard to use in this case (limited size and limited colors) and do not add feature in my opinion.
- I also prefer styled-component but `styled()` makes the app crash when wrapping the Icons... I didn't dig more than that for `styled()`, as I started to review

Reply:
Wait that's strange I use styled(IconButton) both withing ui and fish: https://github.com/MTES-MCT/monitor-ui/blob/main/src/components/SingleTag.tsx#L44 and it works.

And there are 2 props to add whatever color and size you want: https://github.com/MTES-MCT/monitor-ui/blob/main/src/elements/IconButton.tsx#L17

Reply:
Ok, done!